aten/src/ATen/cpu/vec/vec256/vec256_bfloat16.h

```diff
@@ -204,6 +204,12 @@ static_assert(
       return _mm256_loadu_si256(reinterpret_cast<const __m256i*>(ptr));
 
     __at_align__ int16_t tmp_values[size()];
+#ifndef __msvc_cl__
+#pragma unroll
+#endif
+    for (const auto i : c10::irange(size())) {
+      tmp_values[i] = 0;
+    }
```

Comment:
Use memset and initialize things from count to size()?

Reply:
Thanks for your comments. Use `for (const auto i : c10::irange(count, size())) {` instead. It seems that using `std::memset ` does not bring any performance improvement for a small number of data elements.